apps/mappings/helpers.py

```
@@ -1,9 +1,9 @@
 from datetime import datetime, timedelta, timezone
 
 
-from apps.mappings.models import ImportLog
+from fyle_integrations_imports.models import ImportLog
```

Pylint (3.3.7)

[error] Unable to import 'fyle_integrations_imports.models'

(E0401)